frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -512,7 +514,9 @@ export const trendsLogic = kea<
             actions.setBreakdownValuesLoading(false)
         },
         [eventDefinitionsLogic.actionTypes.loadEventDefinitionsSuccess]: async () => {
-            actions.setFilters(getDefaultFilters(values.filters, eventDefinitionsLogic.values.eventNames), true)
+            if (!values.filters?.events?.length && !values.filters?.actions?.length) {
```

Comment:
I think this is the right condition because getDefaultFilters should only be needed when there are no events/actions currently set but I could. be mistaken

Reply:
I think it makes sense, testing

Reply:
I realized this condition is checked inside getDefaultFilters so I changed how we're doing this so there's less repetition